judge/forms.py

```diff
@@ -275,9 +275,13 @@ def clean_handle(self):
             raise ValidationError(_("Account doesn't exist."))
         return self.cleaned_data['handle']
 
-    def send_email(self, pk, token):
+    def send_email(self, user, token):
         url = reverse('wcipeg_merge_activate', args=[pk, token])
-        # TODO: Send email
+        email = EmailMessage(
+            subject='WCIPEG Account Merge Request',
+            message='To authenticate the merge, login as your DMOJ account, then use the link below:\n\n%s' % url,
```

Comment:
Presumably this works even when they're not yet logged in (it asks them to log in first, and then continues the flow after they do so, right?)

Reply:
I'll test but I believe so. I'll also have it automatically log the user out of their WCIPEG account when a merge is requested and clarify the `generic_message` (once more :facepalm:) -- this will happen immediately when the merge is requested.

Reply:
Now that I've taken a second look, I've realized I just confused you and anyone else reviewing. You have to be logged in as the WCIPEG user for the merge request *and* activation because the email is taken from the `request.user.email`. This message is wrong.

This is also the correct method for verifying WCIPEG with password and verifying DMOJ with email.